yt_dlp/extractor/sovietscloset.py

```diff
@@ -0,0 +1,217 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import datetime
+
+from ..utils import js_to_json
+from .common import InfoExtractor
+
+
+class SovietsClosetExtractor(InfoExtractor):
+    MEDIADELIVERY_REFERER = {'Referer': 'https://iframe.mediadelivery.net/'}
+
+    def parse_nuxt_jsonp(self, nuxt_jsonp_url, video_id, name):
+        NUXT_JSONP_RE = r'__NUXT_JSONP__\(.*?\(function\((?P<arg_keys>.*?)\)\{return\s(?P<js>\{.*?\})\}\((?P<arg_vals>.*?)\)'
+        nuxt_jsonp = self._download_webpage(nuxt_jsonp_url, video_id, note=f'Downloading {name} __NUXT_JSONP__')
+
+        js, arg_keys, arg_vals = self._search_regex(NUXT_JSONP_RE, nuxt_jsonp, '__NUXT_JSONP__', group=['js', 'arg_keys', 'arg_vals'])
+
+        arg_keys = arg_keys.split(',')
+        arg_vals = arg_vals.split(',')
+        args = {key: val for key, val in zip(arg_keys, arg_vals)}
+
+        for key, val in args.items():
+            if val in ('undefined', 'void 0'):
+                args[key] = 'null'
+
+        return self._parse_json(js_to_json(js, args), video_id)['data'][0]
+
+    def video_meta(self, video_id, stream_date, game_name, catgory_name, episode_number):
+        title = f'SovietWomble {stream_date} - {game_name}'
+        if catgory_name != 'Misc':
+            title += f' - {catgory_name}'
+        title += f' #{episode_number}'
+
+        stream_date_parsed = datetime.datetime.strptime(f'{stream_date} GMT', '%Y-%m-%d %H:%M %Z')
+        stream_date_yyyymmdd = stream_date_parsed.strftime('%Y%m%d')
+        timestamp = int(stream_date_parsed.timestamp())
```

Comment:
```suggestion
        upload_date = unified_strdate(stream_date)
        timestamp = unified_timestamp(stream_date)
```
from utils.py

the timestamp tests fail now but I don't think it was right to begin with (e.g. for the 'SovietWomble 2017-04-13 13:53 - The Witcher #13' test, your timestamp is giving  2017-04-13 11:53) :confused:

Reply:
This is timezone related. I'm in +02:00 and used my local timezone. I also made the wrong assumption that it's always GMT, but there is summertime, too. Are there any helpers for location based timezones? In this case I'd like to use London.

Reply:
There doesn't seem to be a way to do that without relying on platform specifics or third-party packages, so I just fixed it per your suggestion in https://github.com/yt-dlp/yt-dlp/pull/884/commits/41f0ea806fec6fa402094f1a1161baa83bca800c.